Elasticsearch/Indexer.php

```diff
@@ -91,7 +91,7 @@ public function atomicSwitch(string $alias, string $newIndex, string $removeRege
         array_map([$this, 'delete'], $delete);
     }
 
-    private function getClient(): Client
+    public function getClient(): Client
```

Comment:
I am not sure why you make this functino public? This is not called in your other changes? Revert this back to private.

Reply:
Indeed, I used it before creating a dedicated service. Fixed now.